client/src/store/features/users/usersSlice.js

```diff
@@ -105,11 +106,37 @@ export function unFriend(userToUpdate, id, token) {
     const updatedFriends = updatedUser.friends
 
     /* Find the friend that is in currentFriends but not in updatedFriends */
-    const friendToRemove = currentFriends.find((friend) => (
-      !updatedFriends.some((f) => f.id === friend.id)
-    ))
+    const friendToRemove = currentFriends.find(
+      (friend) => !updatedFriends.some((f) => f.id === friend.id),
+    )
     dispatch(removeFriend(friendToRemove.id))
   }
 }
 
+// todo: consider using the createAsyncThunk middleware
+// async thunk to send delete user request
+export function deleteProfile(userId) {
```

Comment:
Just a reminder here that the delete and put requests to the db requires a valid token

Reply:
You're right, thanks